Basic/DyanmicPrefabs/Assets/Scripts/01_Sparse loading with no latejoin/SparseLoadingNoLatejoinSample.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Threading.Tasks;
+using Unity.Collections;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.UI;
+
+namespace Game
+{
+    /// <summary>
+    /// This is probably the most advanced scenario - we are spawning prefabs that aren't known to clients beforehand.
+    ///
+    /// Here we are not implementing client acknowledgement of the prefab loading completion -
+    /// this would allow the server to wait for all the clients to have loaded the prefab and spawn it only then, which is safer.
+    /// However, this is not implemented here for simplicity - there is a whole slew of potential failures that this would have to handle.
+    ///
+    /// Instead we load the prefab from an addressable, and then send an RPC to the clients, telling them to start loading the prefab.
+    /// Then we wait for a configured amount of time, and then spawn the prefab on the server, which replicates it to the clients.
+    /// Then we hope that the clients will be able to load the prefab during the (m_SpawnDelayInSeconds + m_SpawnTimeoutInSeconds) time window.
```

Comment:
"Then we hope"
I hate that word "hope" (lol --> I sound like a vilain)
Hope is not a strategy. Implementation must be solid if users are to use this in their game